dropwizard-core/src/main/java/io/dropwizard/setup/Environment.java

```diff
@@ -119,6 +119,8 @@ public Environment(String name,
         }
```

Comment:
The file has increased in 2 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._